playground/frontend/build.gradle

```diff
@@ -188,75 +182,54 @@ ext.deleteFilesByRegExp = { re ->
   }
 }
 
+tasks.register("integrationTest") {
+  dependsOn("integrationTest_standalone_change_example")
+}
+
+tasks.register("integrationTest_standalone_change_example") {
+  runIntegrationTest("standalone_change_example", "/")
+}
+
+void runIntegrationTest(String path, String url) {
+  exec {
+    executable("flutter")
+    args(
+      "drive",
+      "--driver=test_driver/integration_test.dart",
+      "--target=integration_test/${path}_test.dart",
+      "--web-launch-url='$url'",
+      "--device-id=chrome",
+    )
+  }
+}
+
 task copyDockerfileDependencies(type: Copy) {
    group = "build"
    description = "Copy files that required to build docker container"
    copy {
-      from '.'
-      into 'build/'
-      exclude 'build'
-      exclude 'Dockerfile'
+      from(".")
+      into("build/")
+      exclude("build")
+      exclude("Dockerfile")
    }
    copy {
-      from '../playground'
-      into 'build/playground'
+      from("../playground")
+      into("build/playground")
    }
 }
 
 docker {
   group = "build"
   description = "Build container for playground frontend application"
-  name containerImageName(
-          name: project.docker_image_default_repo_prefix + "playground-frontend",
-          root: project.rootProject.hasProperty(["docker-repository-root"]) ?
-                  project.rootProject["docker-repository-root"] :
-                  project.docker_image_default_repo_root)
-  files "./build/"
-  tags containerImageTags()
-  buildArgs(['FLUTTER_VERSION': project.rootProject.hasProperty(["flutter-version"]) ?
-                                project.rootProject["flutter-version"] :
-                                "3.3.2" ])
```

Comment:
- No longer allowing to override Flutter version during build. This was unreliable because there are other sources of truth for this as described in #23243